Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -107,8 +107,14 @@
 
 void RemoteLayerTreeDrawingAreaProxy::remotePageProcessCrashed(WebCore::ProcessIdentifier processIdentifier)
 {
-    if (m_remoteLayerTreeHost)
+    if (!m_remoteLayerTreeHost)
+        return;
+
+    if (auto* scrollingCoordinator = m_webPageProxy->scrollingCoordinatorProxy()) {
```

Comment:
Very small nit: other code in this file does `auto scrollingCoordinator = ...`, i.e. without the `*`.

Reply:
We usually prefer `auto*` for raw pointers though.

Reply:
> We usually prefer `auto*` for raw pointers though.

I think it's OK for new code to not match existing one if this is a positive change.